runtime/t0rn-parachain/Cargo.toml

```diff
@@ -43,6 +43,7 @@ pallet-timestamp                           = { git = "https://github.com/parityt
 pallet-transaction-payment                 = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.27" }
 pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.27" }
 pallet-utility                             = { git = "https://github.com/paritytech/substrate", default-features = false, branch = 'polkadot-v0.9.27' }
+pallet-identity                            = { git = "https://github.com/paritytech/substrate", default-features = false, branch = 'polkadot-v0.9.19' }
```

Comment:
same

Reply:
sharp! thanks man. I'll fix that.

Reply:
fixed [b2bfd4a](https://github.com/t3rn/t3rn/pull/459/commits/b2bfd4a99772e8697499671dbe1e6f59f09f997e) thanks!